core/src/main/java/org/apache/iceberg/BaseTable.java

```diff
@@ -39,6 +39,11 @@ public BaseTable(TableOperations ops, String name) {
     this.name = name;
   }
 
+  @Override
+  public String name() {
```

Comment:
Couldn't this method be added in a separate commit?

Reply:
Removed